firedrake/mg/mesh.py

```diff
@@ -201,6 +201,36 @@ def ExtrudedMeshHierarchy(base_hierarchy, layers, kernel=None, layer_height=None
                          refinements_per_level=base_hierarchy.refinements_per_level,
                          nested=base_hierarchy.nested)
 
+def ExtrudedMeshHierarchyUniform(base_hierarchy, base_layers, height, kernel=None,
```

Comment:
Note the change in API: I want to specify the final height of the domain, not the layer height (as this will vary by multigrid level).